training/distributed_training/pytorch/model_parallel/bloom_smp_trainer/submit_smp_trainer.ipynb

```diff
@@ -380,6 +346,9 @@
    "outputs": [],
    "source": [
     "mpioptions = \"-x NCCL_DEBUG=WARN -x SMDEBUG_LOG_LEVEL=ERROR \"\n",
+    "mpioptions += (\n",
+    "    \"-x SMP_DISABLE_D2D=1 -x SMP_D2D_GPU_BUFFER_SIZE_BYTES=1 -x SMP_NCCL_THROTTLE_LIMIT=1 \"\n",
```

Comment:
I tried adding this like other SMP notebooks, but the training job still failed

Reply:
Found the issue, it was because you were overwriting the hyperparameter dict with the dataset update! Will merge and fix.